### PR TITLE
Update teleport container image version to 6.2.7(latest) in teleport-quickstart.yml

### PR DESCRIPTION
**Pull request by flyinghermit**
Trying out teleport quickstart via [docker compose method](https://goteleport.com/docs/setup/guides/docker-compose/#creating-a-teleport-user-when-using-docker-quickstart) fails with error `error: unknown long flag '--roles'` while executing command `docker exec teleport tctl users add testuser --roles=editor,access --logins=root,ubuntu,ec2-user` because the teleport container image version in `teleport-quickstart.yml` still points to v5.

This PR updates the quickstart file with an image version pointing to v6.2.7 (latest)

### CODE REVIEW

File: docker/teleport-quickstart.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ services:
   # The configure container starts, generates a config, writes it to
   # /etc/teleport/teleport.yaml and then immediately exits.
   configure:
-    image: quay.io/gravitational/teleport:5
+    image: quay.io/gravitational/teleport:6.2.7
```
Comment (webvictim) [inline docker/teleport-quickstart.yml:6]:
```suggestion
    image: quay.io/gravitational/teleport:6
```

File: docker/teleport-quickstart.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ services:
   # This container depends on the config written by the configure container above, so it
   # sleeps for a second on startup to allow the configure container to run first.
   teleport:
-    image: quay.io/gravitational/teleport:5
+    image: quay.io/gravitational/teleport:6.2.7
```
Comment (webvictim) [inline docker/teleport-quickstart.yml:17]:
```suggestion
    image: quay.io/gravitational/teleport:6
```

**Review by webvictim — CHANGES_REQUESTED** [review]
If you use `6` as the tag it'll automatically pick the latest 6.x version of Teleport. This saves updating the docs every single time we release.

https://quay.io/repository/gravitational/teleport?tab=tags

These images are rebuilt with the latest version every night.

**Review by webvictim — APPROVED** [review]

**Review by klizhentas — APPROVED** [review]

**Review by russjones — APPROVED** [review]
Bot.

**Pull request closed without merging**